### ISSUE TITLE
The future of this repository

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-design#155]
We plan to move all code from [ckeditor5](https://github.com/ckeditor/ckeditor5) to other packages which means that ckeditor5 repository will be left for general project issues.

Also, we're going to write real documentation for the project finally, so a lot of things covered in this wiki will be moved there.

I start to get worried that ckeditor5 and ckeditor5-design will be competing with each other for attention, leaving confusion among developers.

We need to define what kind of content should be reported and kept where. The distinction should be easy to explain and to enforce. I'd even consider closing ckeditor5-design and moving all the content to ckeditor5. This may clutter ckeditor5 with all the research, prototyping and very general discussions, but at the same time it'll be easier to find all those things, because you'll know where to look for.

**Comment by Reinmar** [comment]
PS. I'd like to make the decision now because later on it will be terribly more costly and confusing. Right now it concerns mostly the core team. Once community kicks in and starts watching some threads it will affect more people.

**Comment by pjasiun** [comment]
Note that `ckeditor5` will be full of bug reports from the community. 

Although, I think that at some point we close the "design" stage, will not introduce so many general changes. At this stage, we should close this repository. Few general changes which appear later should be reported in the `ckeditor5` repo.

**Comment by Reinmar** [comment]
> Note that ckeditor5 will be full of bug reports from the community.

So is http://dev.ckeditor.com and every other issue tracker of a popular OSS project. That doesn't mean you can't have design discussions and decisions made there. It just means you have to tag them correctly. Eventually, they are easier to find if they are all kept in one repo.

> Although, I think that at some point we close the "design" stage, will not introduce so many general changes. At that stage, we should close this repository. Few general changes which appear later should be reported in the ckeditor5 repo.

If we're ever planning to close this repo it should be now.

**Comment by Reinmar** [comment]
BTW. I already started reporting some quite general issues in ckeditor5, such as https://github.com/ckeditor/ckeditor5/issues/246, https://github.com/ckeditor/ckeditor5/issues/245 and https://github.com/ckeditor/ckeditor5/issues/226 because it was natural for me to keep them there. And no one was complaining so far.

I feel that this will be natural for the community as well (as they won't even know ckeditor5-design). I didn't have closing this repository in mind when I've been writing this issue, but the longer I think about this, the more I'm sure this will be a step in the right decision.

**So the question is – does anyone think this is a seriously bad idea to close this repository?**

**Comment by scofalik** [comment]
I do not think it is a bad idea to close this repository, but I wonder how many issues will really be left here after we close outdated ones and convert others to documentation.

I back up @pjasiun on this one. We can have this repo closed later. I don't think we need to worry about mess & confusion until `1.0.0`. But I won't cry if you do it now, either. More important is clearing out outdated issues.

**Comment by Reinmar** [comment]
> I do not think it is a bad idea to close this repository, but I wonder how many issues will really be left here after we close outdated ones and convert others to documentation.

I guess that quite a few. E.g. the UX research. But the bigger problem is how many issues other people will open here that should be in ckeditor5 or in ckeditor5 which should be reported here. Any rule we'll make will be clear only for us.

**Comment by scofalik** [comment]
I see "design" things as big/important changes that have impact on how core team works or are bigger driving factors than just "feature design/idea" (here I mean mobile UX etc.).

Maybe it would be a good idea to have a rule that only core team opens up issues on design? Maybe it's not 100% in the open source spirit, but the community will have ckeditor5 repo and if bigger/more ambitious topic shows out there we can always move it here. WDYT?

**Comment by fredck** [comment]
I would leave this repo opened for now and still keep it as the place of discussion for high level design topics. Meanwhile we create documentation and move development forward. In a future stage we can check things again taking in consideration two possibilities:

 1. The contents and discussions in these repo are happening somewhere else: close it.
 2. People are **constantly** reporting wrong things here so it is clear that it causes confusion: close it.

So far we just have assumptions so let's have them confirmed as it would in any case not hurt us so much. We can close this repo at any moment in the far future without much consequences. We don't need to spent time as well on this at the current stage.

**Comment by scofalik** [comment]
I think that people are more eager to put an issue in repo that looks more general, like ckeditor5. I am afraid, that people will put here issues concerning skins, UI, etc. because that's what they may understand by "design".

**Comment by Reinmar** [comment]
There's one more thing that we're considering now – opening a Disqus for CKEditor 5 which we could use for development news and some general discussions as well. Right now we have no channel where we could post [news and updates](https://github.com/ckeditor/ckeditor5/wiki/News-and-Updates) and stuff like next milestones details and planning. I wanted to post such a news describing next iteration and I simply didn't know where. 

I see how ProseMirror is organised and this is one of the things that I liked. There's Disqus, some live chat and GH. This makes it much clearer what should go where and open more possibilities on what you can post there.

If we'll go this way as well, it means that this repo will become even more redundant.

**Comment by Reinmar** [comment]
OTOH, I try to understand how's Disqus better than GH issues for the job. Seems that the difference is in details, but somehow those small things make it more natural and self-explanatory what you can do there and what you should post using GH issues.

**Comment by oleq** [comment]
I think what you mentioned in your 2 last comments, @Reinmar, is beyond the scope of this issue.

**Comment by Reinmar** [comment]
> I think what you mentioned in your 2 last comments, @Reinmar, is beyond the scope of this issue.

It's strongly related to this repository because it will define the sense of its existence. We shouldn't have too many communication channels. The whole point of this issue was to reduce the number of places where we can report issues. Then I understood that I miss a place where we could post news like "Plans for iteration 2". This make me think that we need such a channel, but that's against the initial point of reducing number of channels.

**Comment by Reinmar** [comment]
To not waste more of your time, I'll just write that after an internal discussion with @wojtekidd we decided to keep `ckeditor5-design`, but we need to seriously work on how it's communicated.

I'll explain the reasoning later on.
